[Stellar] Add toggle to switch from default to explicit trait implementation (#728)
OpenZeppelin/contracts-wizard: .changeset/plain-turkeys-film.md

---
'@openzeppelin/wizard-stellar': patch
'@openzeppelin/wizard-common': patch
'@openzeppelin/contracts-mcp': patch
---

> For stellar add a explicitImplementations flag that switch from using default_impl macro to explicit definitions

Reviewer: ```suggestion
Stellar: add an explicitImplementations flag that switches from using default_impl macro to explicit definitions
```
